### ISSUE TITLE
TypeError when updating webhooks

### ISSUE THREAD

**Issue by alexei** [stripe/stripe-python#591]
Code is basically:
```
stripe.WebhookEndpoint.modify(hook['id'], url=url)
```
Fails with:
```
Traceback (most recent call last):
  [...snip...]
  File "stripe/api_resources/abstract/updateable_api_resource.py", line 12, in modify
    return cls._static_request("post", url, **params)
TypeError: _static_request() got multiple values for keyword argument 'url'
```

That's because [stripe.api_resources.abstract.api_resource.APIResource._static_request](https://github.com/stripe/stripe-python/blob/master/stripe/api_resources/abstract/api_resource.py#L48) accepts a `url` argument which conflicts with the `url` kwarg required for hooks:
```
>>> def foo(bar, **kwargs):
...     pass
... 
>>> foo('a bar', bar='another bar')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: foo() got multiple values for keyword argument 'bar'
```

Fwiw, using stripe==2.31.0 on Python 2.7.10

**Comment by alexei** [comment]
Calling the `.save(url=new_url)` method on the returned hook object seems to work, though.

**Comment by ob-stripe** [comment]
Thanks for the detailed report @alexei! We'll release a fix soon.

**Comment by ob-stripe** [comment]
Fixed in 2.32.1.
